src/thunderbots/software/ai/hl/stp/evaluation/indirect_chip.h

```diff
@@ -0,0 +1,121 @@
+#include "ai/world/world.h"
+#include "geom/angle.h"
+#include "geom/point.h"
+#include "util/parameter/dynamic_parameters.h"
+
+/**
+ * The indirect chip and chase target evaluation function returns a target point for the
+ * chipper and chaser.
+ * Other related evaluation functions are also included in this file.
+ */
+namespace Evaluation
+{
+    typedef Point Vector2;
+    template <size_t N>
+    using Poly     = std::array<Vector2, N>;
+    using Triangle = Poly<3>;
+
+    /**
+     * Returns the target point that the chipper will shoot at and the chaser will meet
+     * ball at. Given the enemy team robot's positions, filters and creates a vector of
+     * triangles to be used as a parameter. The target is where ball will land according
+     * to chipping calibration.
+     *
+     * @param World Object
+     *
+     * @return Point to chip and chase at; If empty, point is not valid
+     */
+    std::optional<Point> indirect_chip_and_chase_target(const World& world);
```

Comment:
perhaps `findPointToTargetForIndirectChipAndChase` might be a bit more descriptive? It's definitely more verbose, but for our team it really makes sense to make things more verbose and clear

Same goes for the other version of the function (that takes triangles and a ball position) below

Reply:
Edited as per suggested